src/Sylius/Behat/Context/Api/Admin/ManagingProductsContext.php

```diff
@@ -0,0 +1,121 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Behat\Context\Api\Admin;
+
+use Behat\Behat\Context\Context;
+use Sylius\Behat\Client\ApiClientInterface;
+use Sylius\Behat\Client\ResponseCheckerInterface;
+use Webmozart\Assert\Assert;
+
+final class ManagingProductsContext implements Context
+{
+    /** @var ApiClientInterface */
+    private $client;
+
+    /** @var ResponseCheckerInterface */
+    private $responseChecker;
+
+    public function __construct(ApiClientInterface $client, ResponseCheckerInterface $responseChecker)
+    {
+        $this->client = $client;
+        $this->responseChecker = $responseChecker;
+    }
+
+    /**
+     * @When I want to create a new configurable product
+     */
+    public function iWantToCreateANewConfigurableProduct(): void
+    {
+        $this->client->buildCreateRequest();
+    }
+
+    /**
+     * @When I specify its code as :code
+     * @When I do not specify its code
+     */
+    public function iSpecifyItsCodeAs(string $code = null): void
+    {
+        $this->client->addRequestData('code', $code);
+    }
+
+    /**
+     * @When I name it :name in :localeCode
+     * @When I rename it to :name in :localeCode
+     */
+    public function iRenameItToIn(string $name, string $localeCode): void
+    {
+        $data = [
+            'translations' => [
+                $localeCode => [
+                    'locale' => $localeCode,
+                    'name' => $name,
+                ],
+            ],
+        ];
+
+        $this->client->updateRequestData($data);
+    }
+
+    /**
+     * @When I set its slug to :slug
+     * @When I set its slug to :slug in :localeCode
+     * @When I remove its slug
+     */
+    public function iSetItsSlugTo(?string $slug = null, $localeCode = 'en_US'): void
+    {
+        $data = [
+            'translations' => [
+                $localeCode => [
+                    'slug' => $slug,
```

Comment:
This should work now:
```suggestion
                    'locale' => $localeCode,
                    'slug' => $slug,
```